docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -37,7 +37,7 @@ All gateway events in Discord are tagged with an opcode that denotes the payload
 | 4011 | Sharding required     | The session would have handled too many guilds - you are required to [shard](#DOCS_TOPICS_GATEWAY/sharding) your connection in order to connect.                                                                                                                 |
 | 4012 | Invalid API version   | You sent an invalid version for the gateway.                                                                                                                                                                                                                                   |
 | 4013 | Invalid intent(s)     | You sent an invalid intent for a [Gateway Intent](#DOCS_TOPICS_GATEWAY/gateway-intents). You may have incorrectly calculated the bitwise value.                                                                                                                               |
-| 4014 | Disallowed intent(s)  | You sent a disallowed intent for a [Gateway Intent](#DOCS_TOPICS_GATEWAY/gateway-intents). You may have tried to specify an intent that you [have not enabled or are not whitelisted for](#DOCS_TOPICS_GATEWAY/privileged-intents).              |
+| 4014 | Disallowed intent(s)  | You sent a disallowed intent for a [Gateway Intent](#DOCS_TOPICS_GATEWAY/gateway-intents). You may have tried to specify an intent that you [have not enabled or are not allowed to use](#DOCS_TOPICS_GATEWAY/privileged-intents).              |
```

Comment:
Same as above. This is now redundant. "A disallowed intent is one which you have not enabled for your bot or one that your bot is not allowed to use." -> "You are disallowed to use a disallowed intent." It should explain why you are disallowed.